x-pack/plugins/enterprise_search/public/applications/app_search/components/result_field_value/result_field_value.scss

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+.enterpriseSearchDataType {
+  &--number,
+  &--float {
+    color: #e53959;
```

Comment:
@constancecchen Any idea what colors to use here?

I just ported these colors over from our old UI. They cause axe failures (see the following) and I'm sure they don't follow whatever EUI standards they may be.

> Element has insufficient color contrast of 4.14 (foreground color: #e53959, background color: #ffffff, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1

Are there colors we could / should use from a predefined color palette? And if so, are there shared sass variables we can reference?

Reply:
See the screenshot on the main PR ... the green and red text.

Reply:
https://github.com/elastic/kibana/pull/83564/commits/7a19a69e1829af49dc3addaee7af207bcdd7fcf3